### PR TITLE
feat: add new nextsteps admin to mailchimp audience

### PR DESCRIPTION
**Pull request by Kneesal**
# Description

Automatically upserts user's email to Mailchimp via it's marketing API when they sign up 

- Link to Basecamp Todo

# How should this PR be QA Tested?

Please describe the QA tests that you ran to verify your changes. Please also note any relevant details for your test configuration.

- [ ] newly created users email address should be entered in the contacts list for mailchimp audience 

# Walkthrough

copilot:walkthrough

### CODE REVIEW

**Review by tataihono — CHANGES_REQUESTED** [review]

File: apps/api-users/src/app/modules/mailChimp/mailChimp.service.spec.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    lists: {
+      ...originalModule.lists,
+      setListMember: jest.fn()
+    }
+  }
+})
+
+describe('MailChimpService', () => {
+  let mailChimpService: MailChimpService
+  ;(mailchimp.lists.setListMember as jest.Mock).mockResolvedValue({})
```
[the remark this answers is not in the document]
Comment (Kneesal) [reply apps/api-users/src/app/modules/mailChimp/mailChimp.service.spec.ts:20]: apologies, this must have been a mistake

**Review by tataihono — APPROVED** [review]

**Review by tataihono — DISMISSED** [review]

**Comment by tataihono** [comment]
atlantis plan

**Review by tataihono — DISMISSED** [review]

**Comment by tataihono** [comment]
atlantis plan

**Comment by tataihono** [comment]
atlantis plan

**Comment by tataihono** [comment]
atlantis apply

**Review by tataihono — APPROVED** [review]

**Review by mikeallisonJS — APPROVED** [review]

**Comment by Kneesal** [comment]
atlantis unlock

**Pull request merged**